packages/@aws-cdk/assert-internal/jest.ts

```diff
@@ -37,7 +37,6 @@ expect.extend({
     actual: cxapi.CloudFormationStackArtifact | core.Stack,
     template: any,
     matchStyle?: MatchStyle) {
-
```

Comment:
This is a newline I love. I would hate to see it go :(

Reply:
i think this could be replaced by indenting function declaration parameters so they dont align with the next block